Newsletter for Oct 2019 (#713)
alan-turing-institute/the-turing-way: communications/newsletters/newsletter_02_Oct2019.md

# The Turing Way Newsletter: 04 October 2019

Hello Turing Way friends!

This month we have been drinking virtual coffees at Calleboration Cafes :coffee: and discussing with you how you can contribute to The Turing Way project.

We ran **4 Online Collaboration Cafes** ([videos](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA)) and partnered with the Mozilla Open Leader X - [Open Life Science mentoring program](https://openlifesci.org).

Shout out to Sarah Gibson and Rachael Ainsworth who gave talks and ran a workshop on The Turing Way Project. Thanks to Paula Andrea Martinez and Philip Durbin for engaging with the project from Australia and USA. 

We are excited to meet many of you at [MozFest](https://www.mozillafestival.org/en/) in London, and have been thinking of ways to involve people in different time zones.

Find more details on these topics below. :point_down:

See you after Halloween! :jack_o_lantern: :maple_leaf:

## Community meetings

### Next Collaboration Cafes

> This month’s collaboration cafes will take place on 2 October at 15:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-02/15:00/TuringWay-CollaborationCafe)), 16 October at 19:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-16/19:00/TuringWay-CollaborationCafe)), and 30 October at 08:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-30/08:00/TuringWay-CollaborationCafe)). We would love to have you on these calls to continue working together on different GitHub issues or help you get started if you are new to the project. You can indicate your participation by signing up on this HackMD: [https://hackmd.io/@KirstieJane/CollabCafe](https://hackmd.io/@KirstieJane/CollabCafe). 

Reviewer: ```suggestion
This month’s collaboration cafes will take place on 2 October at 15:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-02/15:00/TuringWay-CollaborationCafe)), 16 October at 19:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-16/19:00/TuringWay-CollaborationCafe)), and 30 October at 08:00 GMT ([your local time](https://arewemeetingyet.com/London/2019-10-30/08:00/TuringWay-CollaborationCafe)).
We would love to have you on these calls to continue working together on different GitHub issues or help you get started if you are new to the project.
You can indicate your participation by signing up on this HackMD: [https://hackmd.io/@KirstieJane/CollabCafe](https://hackmd.io/@KirstieJane/CollabCafe), or come and say hello in our Gitter channel: [https://gitter.im/alan-turing-institute/the-turing-way](https://gitter.im/alan-turing-institute/the-turing-way).
```

Reviewer 2: The first one now has already happened, do you want/need to reword?

Author: Ah, true. I will edit that now.

### MozFest 2019

> Several community members of The Turing Way will be attending [Mozilla Festival 2019](https://www.mozillafestival.org/en/) in London from 21 to 27 October. If you are in town and also attending this conference, get in touch through our [Gitter channel](https://gitter.im/alan-turing-institute/the-turing-way) or other communication channels listed below this newsletter. We look forward to seeing you there in person.

Author: Since it is not an open meeting, we can leave that out for the next newsletter with the updates and future plans. What do you think?

## News from the community

### Collaboration Cafes online call series

> Ever since it's launch in September 2019, already 3 online Collaboration Cafes have taken place on 4th, 18th and 19th September 2019. Thanks to all our attendees for coming along and joining the calls to learn more about these calls, tips on community management, GitHub tricks and the project itself. You can watch the recording from these calls on our [YouTube channel](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA).

Reviewer: ```suggestion
Ever since their launch in September 2019, 3 online Collaboration Cafes have already taken place on 4th, 18th and 19th September 2019.
Thanks to all our attendees for coming along and joining us to learn more about these calls, tips on community management, GitHub tricks and the project itself.
You can watch the recording from these calls on our [YouTube channel](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA).
```

> Read all about this online call series on [this page](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md). We are still developing the format (see the [template](https://github.com/alan-turing-institute/the-turing-way/blob/master/communications/collaboration-cafe/collaboration-cafe-template.md)) and will be happy to get your feedback to make these sessions useful for you.

Reviewer: ```suggestion
Read all about this online call series on [this page](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md).
We are still developing the format (see the [template](https://github.com/alan-turing-institute/the-turing-way/blob/master/communications/collaboration-cafe/collaboration-cafe-template.md)) and will be happy to get your feedback to make these sessions useful for you.
(For example, we've already added an "Australia-friendly" cafe on 30 October because Paula [got in touch](https://github.com/alan-turing-institute/the-turing-way/issues/684) 💖)
There's an issue open to collected your ideas ([#711](https://github.com/alan-turing-institute/the-turing-way/issues/711)).
Thank you in advance!
```

> #### Online calls for different time zones

Reviewer: I think we can fold this information into the earlier section - what do you think?

## Tips & Tricks for new contributors

Are you new to this project and want to participate in the community? You can start by reading the [contribution guideline](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md), where you can get an overview of the project and the different communication channels you can use to participate.

See [this page](https://github.com/alan-turing-institute/the-turing-way) to find out who is in the community and check out this list of ['good first pull' issues](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20issue) to get to know this project well.

## Acknowledgements and celebrations!

Shout out to two of our members [Sarah Gibson](https://twitter.com/drsarahlgibson) and [Rachael Ainsworth](https://twitter.com/rachaelevelyn), who represented The Turing Way project at two separate conferences in September.

> Sarah, Research Data Scientist at The Alan Turing Institute, delivered a successful workshop on building a [BinderHub](https://binderhub.readthedocs.io/en/latest/) and The Turing Way at the [RSEconUK 2019](https://rse.ac.uk/conf2019/). She shared her training materials [online on Zenodo](https://zenodo.org/record/3404774#.XZN7h5MzbpB), under the DOI: 10.5281/zenodo.3404774.

Reviewer: ```suggestion
Sarah, Research Data Scientist at The Alan Turing Institute, delivered a successful workshop on building a [BinderHub](https://binderhub.readthedocs.io/en/latest/) and _The Turing Way_ at the [RSEconUK 2019](https://rse.ac.uk/conf2019/).
She shared her training materials in the [GitHub repository](https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md) and online via Zenodo (DOI: [10.5281/zenodo.3382208](https://doi.org/10.5281/zenodo.3382208)).
```

> Rachael, Community Manager of Software Sustainability Institute, attended the Open Science Fair 2019 in Porto, Portugal, last month, where she presented a poster and a [demo session](https://www.opensciencefair.eu/demos-2019/the-turing-way-a-handbook-for-reproducible-data-science) on The Turing Way project. See her [tweet-thread](https://twitter.com/rachaelevelyn/status/1173542269355532289) from the conference.

Reviewer: ```suggestion
Rachael, Community Manager of Software Sustainability Institute, attended the Open Science Fair 2019 in Porto, Portugal, last month, where she presented a poster and a [demo session](https://www.opensciencefair.eu/demos-2019/the-turing-way-a-handbook-for-reproducible-data-science) on _The Turing Way_ project.
See her [tweet-thread](https://twitter.com/rachaelevelyn/status/1173542269355532289) from the conference, her demo presentation (DOI: [10.5281/zenodo.3403160](https://doi.org/10.5281/zenodo.3403160)) and poster presentation (DOI: [10.5281/zenodo.3381445](https://doi.org/10.5281/zenodo.3381445))!.
```

Reviewer 2: Should we plug the podcast? I think that would be quite nice. https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/

Author: yes. Of course. I missed that.

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Hi <a href="https://twitter.com/hashtag/OSFair2019?src=hash&amp;ref_src=twsrc%5Etfw">#OSFair2019</a>! 👋 I’ve got a poster presenting the <a href="https://twitter.com/hashtag/TuringWay?src=hash&amp;ref_src=twsrc%5Etfw">#TuringWay</a>: an open &amp; collaborative project to develop a handbook for reproducible data science. Feel free to come chat to me about this project (or anything <a href="https://twitter.com/hashtag/OpenScience?src=hash&amp;ref_src=twsrc%5Etfw">#OpenScience</a> related!) at poster stand #80. <a href="https://t.co/tMAXVn2mKq">pic.twitter.com/tMAXVn2mKq</a></p>&mdash; Dr. Rachael Ainsworth (@rachaelevelyn) <a href="https://twitter.com/rachaelevelyn/status/1173625329686110208?ref_src=twsrc%5Etfw">September 16, 2019</a></blockquote> 

## Connect with us!

- [About the project](https://www.turing.ac.uk/research/research-projects/turing-way-handbook-reproducible-data-science)

> - [The Turing Way Book](https://the-turing-way.netlify.com)

Reviewer: ```suggestion
- [_The Turing Way_ book](https://the-turing-way.netlify.com)
```

> - [GitHub](https://github.com/alan-turing-institute/the-turing-way)

Reviewer: ```suggestion
- [GitHub repository](https://github.com/alan-turing-institute/the-turing-way)
```

> - [Gitter](https://gitter.im/alan-turing-institute/the-turing-way)

Reviewer: ```suggestion
- [Gitter chat room](https://gitter.im/alan-turing-institute/the-turing-way)
```

- [YouTube Videos](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA)
- Twitter Hashtag [#TuringWay](https://twitter.com/hashtag/TuringWay?f=live)

You are welcome to contribute contents for the next newsletter by by

> emailing the project coordinator [Kirstie Whitaker](mailto:kwhitaker@turing.ac.uk).

Reviewer: Ah - this is not a great mechanism - I'm terrible with email 😬 

Can they email you @malvikasharan? Or can we make an issue?

Fine to leave it as me if that's the best option....

Author: Yes, of course :) They can email me in my turing email id, msharan@turing.ac.uk

Reviewer: Thank you so much 🙏

Author: Updated the newsletter with your comments. Do you have any comment on the format?

> *Did you miss the last Newsletters? Check them [here](https://github.com/alan-turing-institute/the-turing-way/blob/master/communications/newsletters/newsletter_06_Sept2019.md)*.

Reviewer: ```suggestion
*Did you miss the last Newsletters? Check them out [here](https://tinyletter.com/TuringWay/archive)*.
```

There's an issue open to collected your ideas ([#711](https://github.com/alan-turing-institute/the-turing-way/issues/711)).
Thank you in advance!

## Relevant resources

### Open Life Science Mentoring Program

The Turing Way is delighted to collaborate with the [Open Life Science Program](https://openlifesci.org) organised by Bérénice Batut, Yo Yehudi and Malvika Sharan as one of the [Mozilla Open Leader X projects](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/). They are developing this mentoring program for early-stage researchers who want to become ambassadors for Open Life Science in their communities. A call for application for mentees and mentors will be announced soon. Stay up to date through their webpage (https://openlifesci.org/).

## Tips & Tricks for new contributors

Are you new to this project and want to participate in the community? You can start by reading the [contribution guideline](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md), where you can get an overview of the project and the different communication channels you can use to participate.

> See [this page](https://github.com/alan-turing-institute/the-turing-way) to find out who is in the community and check out this list of ['good first pull' issues](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20issue) to get to know this project well.

Reviewer: ```suggestion
See [this page](https://github.com/alan-turing-institute/the-turing-way) to find out who is in the community and check out this list of [good first issues](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20issue) to get to know this project well.
```
Just removed pull if that's ok because I think there are lots of ways of contributing without making a PR...

## Acknowledgements and celebrations!

Shout out to two of our members [Sarah Gibson](https://twitter.com/drsarahlgibson) and [Rachael Ainsworth](https://twitter.com/rachaelevelyn), who represented The Turing Way project at two separate conferences in September.

Sarah, Research Data Scientist at The Alan Turing Institute, delivered a successful workshop on building a [BinderHub](https://binderhub.readthedocs.io/en/latest/) and _The Turing Way_ at the [RSEconUK 2019](https://rse.ac.uk/conf2019/).

> She shared her training materials in the [GitHub repository](https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md) and online via Zenodo (DOI: [10.5281/zenodo.3382208](https://doi.org/10.5281/zenodo.3382208)).

Reviewer: Do we have a good picture of @sgibson91 from RSECon?

Author: <blockquote class="twitter-tweet"><p lang="en" dir="ltr">.<a href="https://twitter.com/drsarahlgibson?ref_src=twsrc%5Etfw">@drsarahlgibson</a> running through a brilliant intro at the Build <a href="https://twitter.com/mybinderteam?ref_src=twsrc%5Etfw">@mybinderteam</a> Hub workshop at <a href="https://twitter.com/hashtag/ukrse19?src=hash&amp;ref_src=twsrc%5Etfw">#ukrse19</a>.<br><br>Learned loads at the first one she ran through <a href="https://twitter.com/hashtag/turingway?src=hash&amp;ref_src=twsrc%5Etfw">#turingway</a> and it&#39;s even more informative this time round! 🤩 <a href="https://t.co/hcCfuIaxZ1">pic.twitter.com/hcCfuIaxZ1</a></p>&mdash; annakrystalli (@annakrystalli) <a href="https://twitter.com/annakrystalli/status/1174603229679763457?ref_src=twsrc%5Etfw">September 19, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

# The Turing Way Newsletter: 04 October 2019

Hello Turing Way friends!

This month we had virtual coffees at **4 Online Collaboration Cafes** ([videos](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA), [more info](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md)) :coffee: and discussed with you how you can contribute to this project.

> _The Turing Way_ was highlighted in the [The Turing impact story](https://www.turing.ac.uk/research/impact-stories/transforming-culture-data-science) authored by [Sean O'Neill](https://www.turing.ac.uk/people/business-team/sean-oneill) under the title **Transforming the culture of data science**.

Reviewer: 😍 😍

Shout out to Sarah Gibson and Rachael Ainsworth who gave talks and ran a workshop/demo-sesson on _The Turing Way_ Project. 
Thanks to Paula Andrea Martinez and Philip Durbin for engaging with the project from Australia and USA 🌎 🌏 🌍 

Say hello to [Malvika Sharan](https://twitter.com/MalvikaSharan), who joined _The Turing Way_ team as a Community Manager (and is drafting this newsletter :blush:).

We are excited to meet many of you in London at [MozFest 2019](https://www.mozillafestival.org/en/). 

That's all from us before Halloween! :jack_o_lantern: :maple_leaf:

Find more details on these topics below. :point_down:

## Community meetings

### Next Collaboration Cafes: online call series

> This month’s collaboration cafes are scheduled on 2 October at 15:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-02/15:00/TuringWay-CollaborationCafe)), 16 October at 19:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-16/19:00/TuringWay-CollaborationCafe)), and 30 October at 08:00 GMT ([your local time](https://arewemeetingyet.com/London/2019-10-30/08:00/TuringWay-CollaborationCafe)).

Reviewer: Do we need to rephrase this now that the first one has passed? Or is it ok? I'm in two minds!

Update: I think it make sense to be clear that one has passed....

```suggestion
We have three collaboration cafes this month!
The first was on 2 October at 15:00 BST and the next two are on 16 October at 19:00 BST ([your local time](https://arewemeetingyet.com/London/2019-10-16/19:00/TuringWay-CollaborationCafe)), and 30 October at 08:00 GMT ([your local time](https://arewemeetingyet.com/London/2019-10-30/08:00/TuringWay-CollaborationCafe)).
```

> You can indicate your participation by signing up on this HackMD: [https://hackmd.io/@KirstieJane/CollabCafe](https://hackmd.io/@KirstieJane/CollabCafe), or come and say hello in our Gitter channel: [https://gitter.im/alan-turing-institute/the-turing-way](https://gitter.im/alan-turing-institute/the-turing-way). We would love to have you on these calls to continue working together on different GitHub issues or help you get started if you are new to the project.

Reviewer: ```suggestion
You can indicate your participation by signing up on this HackMD: [https://hackmd.io/@KirstieJane/CollabCafe](https://hackmd.io/@KirstieJane/CollabCafe), or come and say hello in our Gitter channel: [https://gitter.im/alan-turing-institute/the-turing-way](https://gitter.im/alan-turing-institute/the-turing-way).
We would love to have you on these calls to continue working together on different GitHub issues or help you get started if you are new to the project.
```

### MozFest 2019

Several community members of _The Turing Way_ will be attending [Mozilla Festival 2019](https://www.mozillafestival.org) in London from 21 to 27 October.

> If you are in town and also attending this conference, get in touch through our [Gitter channel](https://gitter.im/alan-turing-institute/the-turing-way) or other communication channels listed below this newsletter. We look forward to seeing you there in person.

Reviewer: ```suggestion
Kirstie and Malvika will be co-working at MozFest House on **Thursday 24 October** from 10am to 4pm.
If you are in town we would love to see you there for any amount of time!
To let the festival team plan appropriately, please sign up for the [Co-Working Space @ Mozfest House](https://www.mozillafestival.org/en/house/thursday/) (scroll to almost the end of the list of events to find that option, click and then grab a free ticket for Thursday) and then come along and say hello!

We look forward to seeing you in person!
```

## News from the community

![](https://www.turing.ac.uk/sites/default/files/inline-images/Reproducibility.jpg)
*The Turing Way illustration published in the imapct story, art by Scriberia, CC-BY licence.*

### Update on the last Collaboration Cafes

Ever since their launch in September 2019, total 4 online Collaboration Cafes have already taken place. :tada:

Thanks to all our attendees for coming along and joining us to learn more about these calls, tips on community management, GitHub tricks and the project itself. Read all about this online call series on [this page](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md) and watch the recording from previous calls on our [YouTube channel](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA). 

> Get in touch with us, if you would like to join a call in the future and if our [current schedule](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md#attending-an-online-collaboration-cafe) is not compatible with your time zone. Thanks to [Paula Andrea Martinez](https://github.com/orchid00), National Characterisation Training Coordinator at the Australian National Imaging Facility, we've already added an "Australia-friendly" slot on 30 October. :sparkling_heart:

Reviewer: ```suggestion
Get in touch with us, if you would like to join a call in the future and if our [current schedule](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md#attending-an-online-collaboration-cafe) is not compatible with your time zone.
Thanks to [Paula Andrea Martinez](https://github.com/orchid00), National Characterisation Training Coordinator at the Australian National Imaging Facility, we've already added an "Australia-friendly" slot on 30 October. :sparkling_heart:
```

> We are still developing the format (see the [template](https://github.com/alan-turing-institute/the-turing-way/blob/master/communications/collaboration-cafe/collaboration-cafe-template.md)) and will be happy to get your feedback to make these sessions useful for you. There's a GitHub issue [#711](https://github.com/alan-turing-institute/the-turing-way/issues/711) open for your comments and ideas.

Reviewer: ```suggestion
We are still developing the format (see the [template](https://github.com/alan-turing-institute/the-turing-way/blob/master/communications/collaboration-cafe/collaboration-cafe-template.md)) and will be happy to get your feedback to make these sessions useful for you.
There's a GitHub issue [#711](https://github.com/alan-turing-institute/the-turing-way/issues/711) open for your comments and ideas.
```

Thank you in advance!

## Relevant resources

### _The Turing Way_ Impact Story

> _The Turing Way_ project was highlight among [The Turing Impact Stories](https://www.turing.ac.uk/research/impact-stories) this month. Under the title [Transforming the culture of data science](https://www.turing.ac.uk/research/impact-stories/transforming-culture-data-science), [Sean O'Neill](https://www.turing.ac.uk/people/business-team/sean-oneill) highlighted its role in representing the institute’s commitment to changing data science for the better. Several community members including the project lead [Kirstie Whitaker](https://www.turing.ac.uk/people/researchers/kirstie-whitaker) shared their journey and impact related to this project. You can read the main points in this 1-page short [PDF summary](https://www.turing.ac.uk/sites/default/files/2019-10/impact_story_-_transforming_the_culture_of_data_science.pdf).

Reviewer: ```suggestion
_The Turing Way_ project was highlight among [The Turing Impact Stories](https://www.turing.ac.uk/research/impact-stories) this month.
Under the title [Transforming the culture of data science](https://www.turing.ac.uk/research/impact-stories/transforming-culture-data-science), [Sean O'Neill](https://www.turing.ac.uk/people/business-team/sean-oneill) highlighted its role in representing the institute’s commitment to changing data science for the better.
Several community members including the project lead [Kirstie Whitaker](https://www.turing.ac.uk/people/researchers/kirstie-whitaker) shared their journey and impact related to this project.
You can read the main points in this 1-page short [PDF summary](https://www.turing.ac.uk/sites/default/files/2019-10/impact_story_-_transforming_the_culture_of_data_science.pdf).
```

> "The moonshot goal of the project is to make reproducible research too easy not to do."

Please share it with people who might be interested in joining our community and contributing to the project. :gift_heart:

## Tips & Tricks for new contributors

> Are you new to this project and want to participate in the community? You can start by reading the [contribution guideline](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md), where you can get an overview of the project and the different communication channels you can use to participate.

Reviewer: ```suggestion
Are you new to this project and want to participate in the community?
You can start by reading the [contribution guideline](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md), where you can get an overview of the project and the different communication channels you can use to participate.
```

See [this page](https://github.com/alan-turing-institute/the-turing-way) to find out who is in the community and check out this list of [good first issues](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20issue) to get to know this project well.

## Acknowledgements and celebrations!

> Shout out to two of our members [Sarah Gibson](https://twitter.com/drsarahlgibson) and [Rachael Ainsworth](https://twitter.com/rachaelevelyn), who represented The Turing Way project at two separate conferences in September.

Reviewer: ```suggestion
Shout out to two of our members [Sarah Gibson](https://twitter.com/drsarahlgibson) and [Rachael Ainsworth](https://twitter.com/rachaelevelyn), who represented _The Turing Way_ at two separate conferences in September.
```

## Connect with us!

- [About the project](https://www.turing.ac.uk/research/research-projects/turing-way-handbook-reproducible-data-science)
- [_The Turing Way_ book](https://the-turing-way.netlify.com)
- [GitHub repository](https://github.com/alan-turing-institute/the-turing-way)
- [Gitter chat room](https://gitter.im/alan-turing-institute/the-turing-way)
- [YouTube Videos](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA)
- Twitter Hashtag [#TuringWay](https://twitter.com/hashtag/TuringWay?f=live)

You are welcome to contribute contents for the next newsletter by
emailing [Malvika Sharan](mailto:msharan@turing.ac.uk).

> *Did you miss the last Newsletters? Check them out [here](https://tinyletter.com/TuringWay/archive)*.

Reviewer: ```suggestion
*Did you miss the last newsletters?*
*Check them out [here](https://tinyletter.com/TuringWay/archive)*.
```
